src/commons/api/roas.rs

```diff
@@ -280,22 +286,186 @@ impl Ord for RoaDefinition {
     }
 }
 
-impl PartialOrd for RoaDefinition {
+impl PartialOrd for RoaPayload {
     fn partial_cmp(&self, other: &Self) -> Option<Ordering> {
         Some(self.cmp(other))
     }
 }
 
-impl AsRef<TypedPrefix> for RoaDefinition {
+impl AsRef<TypedPrefix> for RoaPayload {
     fn as_ref(&self) -> &TypedPrefix {
         &self.prefix
     }
 }
 
+//------------ RoaConfiguration --------------------------------------------
+
+/// This type defines a saved RoaConfiguration. It includes the actual ROA
```

Comment:
> This type defines a saved RoaConfiguration
Perhaps good to avoid a self-referential description of the type, it's a bit strange for the reader. "An apple is an apple" ...

Reply:
fixed